evm/gateway.js

```diff
@@ -183,6 +195,38 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'approveWithBatch': {
+            const { batchID, api } = options;
+
+            if (!batchID) {
+                throw new Error('Batch ID is required for the approve action');
+            }
+
+            const batchId = batchID.startsWith('0x') ? batchID.substring(2) : batchID;
+            let apiUrl = api || `${config.axelar.lcd}/axelar/evm/v1beta1/batched_commands/${chain.name.toLowerCase()}`;
+            apiUrl = apiUrl.endsWith('/') ? apiUrl.slice(0, -1) : apiUrl;
```

Comment:
this logic is an example of duplication due to a lack of a strong guarantee, and every usage of the api will cause this. Maybe add a test for the info files here to check it's not added: https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/tests/info.test.js#L20

we could probably use trimPrefix, trimSuffix helpers as well in our utils